test/unit/metrics_plots/conftest.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
I'd rather have these bits in the test files themselves.

Reply:
But that would mean we need to duplicate it if there are multiple files. Or did you mean "make it all a single file"? I'll go with that for now, but lmk if you meant something else.

Reply:
I meant duplicating it in the file. It makes understanding a test file as a standalone file much easier. I'm not sure how common it is to load these datasets in the conftest, but I'm not used to it. And for a third person adding a test in a test file, they'd be a bit confused or not see that there are datasets available and they'd load their own in the tests.

Reply:
That's fair. We've been doing it like that throughout the test/unit directory so far, so that's probably why Kevin originally copied this behavior. I assume the current structure is in line with your expectations. If not, please let me know!